### ISSUE TITLE
[Bug] Upon deleting Grid, Board or Calendar from the sidebar, no banner to restore or delete permanently appears.

### ISSUE THREAD

**Issue by MayurSMahajan** [AppFlowy-IO/AppFlowy#3471]
### Bug Description

When you delete a document in AppFlowy you see a dialog appear at the top of your document:
![image](https://github.com/AppFlowy-IO/AppFlowy/assets/47064215/09ddb975-bb25-41d3-be1d-3246bb8ecc12)

No such dialog appears when you delete a Grid, Board or Calendar. Since there is no indicator of the file being deleted, the user may continue editing the Grid/Board/Calendar without noticing that they have deleted it. 

This is how it appears:

https://github.com/AppFlowy-IO/AppFlowy/assets/47064215/908507cb-8010-4a6e-a758-578818738c00

### How to Reproduce

1. Open AppFlowy
2. Create a new Grid/Board/Calendar
3. Delete the newly created item from the sidebar
4. Observe how the deleted item still appears without any indicator of it being deleted.
5. Observe how you can still change the deleted item's content.

### Expected Behavior

Some indication that the Grid/Board/Calendar is deleted and thus should not be modified.

### Operating System

Windows 11

### AppFlowy Version(s)

1.0.0

### Screenshots

Refer to the video attached above.

**Comment by Azanul** [comment]
May I take this up?

**Comment by richardshiue** [comment]
Assigned @Azanul, have fun!

**Comment by 0xJaskeerat** [comment]
Hey @MayurSMahajan @richardshiue  , Can I try my hands on this , if @Azanul is not able to do

**Comment by MayurSMahajan** [comment]
@Azanul are you still working on this?

**Comment by Azanul** [comment]
> @Azanul are you still working on this?

Yes. I haven't used block before that's why I'm taking some time, but I'll raise the PR in a couple of days.

**Comment by SharmaDhruv2511** [comment]
is this issue still open??

**Comment by Xazin** [comment]
> is this issue still open??

There is an open PR for this, thus it's not closed yet, but work are done/in progress.

**Comment by rileyhawk1417** [comment]
This bug is still there but on mobile this time. Specifically for the "document" type.
![Screenshot_20250115-224109.jpg](https://github.com/user-attachments/assets/718ded0b-f0ef-42d8-b676-0ecf2f8b8466)


The delete banner shows at the top of the screen.

**Comment by rileyhawk1417** [comment]
This bug is still there but on mobile this time. Specifically for the "document" type.
![Screenshot_20250115-224109.jpg](https://github.com/user-attachments/assets/718ded0b-f0ef-42d8-b676-0ecf2f8b8466)


The delete banner shows at the top of the screen.

**Comment by appflowy** [comment]
This issue has already been fixed and will be included in the upcoming release (0.10.7). I’m closing this issue.
